pkg/cosign/cosign.go

```diff
@@ -591,7 +595,7 @@ func getCTLogPubs(ctx context.Context, ctlogPubKey string) (*cosign.TrustedTrans
 
 	publicKeys := cosign.NewTrustedTransparencyLogPubKeys()
 	if err := publicKeys.AddTransparencyLogPubKey([]byte(ctlogPubKey), tuf.Active); err != nil {
-		return nil, fmt.Errorf("AddRekorPubKey: %w", err)
+		return nil, fmt.Errorf("AddCTLogsPubKey: %w", err)
```

Comment:
Please improve the error message:

`failed to get transparency log public keys`